server/routerlicious/packages/services-client/src/restWrapper.ts

```diff
@@ -129,7 +129,7 @@ export class BasicRestWrapper extends RestWrapper {
         return new Promise<T>((resolve, reject) => {
             this.axios.request<T>(options)
                 .then((response) => { resolve(response.data); })
-                .catch((error: AxiosError) => {
+                .catch((error: AxiosError<any>) => {
```

Comment:
Same change we applied [when upgrading to Axios 1.x in main](https://github.com/microsoft/FluidFramework/pull/17419/files#diff-6ad2b284866501b71c805cb9805e1077821ab8172b0c513abe1fe8d90fdf0b30R172).